internal/template/template.go

```diff
@@ -44,6 +45,34 @@ func GenerateVarsTf(opts VarsOpts, path string) error {
 	return nil
 }
 
+func GenerateConfigFile(opts ConfigOpts) error {
+	configPath, _ := os.Getwd()
+
+	f := hclwrite.NewEmptyFile()
+
+	rootBody := f.Body()
+
+	rootBody.SetAttributeValue("env", cty.StringVal(opts.ENV))
+	rootBody.SetAttributeValue("aws_profile", cty.StringVal(opts.AWS_PROFILE))
+	rootBody.SetAttributeValue("aws_region", cty.StringVal(opts.AWS_REGION))
+	rootBody.SetAttributeValue("terraform_version", cty.StringVal(opts.TERRAFORM_VERSION))
+	rootBody.SetAttributeValue("namespace", cty.StringVal(opts.NAMESPACE))
+
+	file, err := os.Create(fmt.Sprintf("%s/%s", configPath, ize))
+	if err != nil {
+		return err
+	}
+
```

Comment:
Question: can we create commented sections? So, for example, can we generate the following block (with comments):

```hcl
/*
# ECS Service Example
service "ecs" "myservice" {
   tasks_min = 1
   tasks_max = 2
   tasks_desired = 2
}
*/
```